apps/docs/pages/guides/getting-started/tutorials/with-react.mdx

```diff
@@ -378,8 +378,7 @@ return (
       url={avatar_url}
       size={150}
       onUpload={(event, url) => {
-        setAvatarUrl(url)
-        updateProfile(event)
+        updateProfile(event, url)
```

Comment:
We probably want to keep `setAvatarUrl(url)` so that the avatar updates with the new image right? I could be missing something.

Reply:
@gregnr 

> the fallback avatar_url should be unnecessary right?
Yeah, makes sense!

> so that the avatar updates with the new image right?
I think it would be better to update state inside the `updateProfile()` function. That way we avoid showing the avatar update, if updating the profile has failed. WDYT?